ush/parsing_namelists_FV3.sh

```diff
@@ -17,6 +17,10 @@ if (( gwd_opt == 2 )) && [[ ${do_gsl_drag_ls_bl} == ".true." ]]; then
   cdmbgwd=${cdmbgwd_gsl}
 fi
 
+if (( OFFSET_START_HOUR != 0 )); then
+  DIAG_TABLE="${PARMgfs}/ufs/fv3/diag_table_replay"
+fi
```

Comment:
Please set this in `config.fcst`

Reply:
This has been moved to `parm/config/gefs/config.fcst`.